src/workloads/encrypted/qe-range-age-100-0.yml

```diff
@@ -0,0 +1,1036 @@
+SchemaVersion: 2018-07-01
+Owner: "@10gen/server-security"
+Description: |
+  QE Range Release Criteria Experiment
+Keywords:
+- Queryable Encryption
+
+GlobalDefaults:
+  MyDatabase: &encrypted_db genny_release_criteria
+  MyCollection: &encrypted_coll my_coll
+  NopAlias: &nop {Nop: true}
+  
+  StateTransitions: &transitions
+  - To: Query
+    Weight: 1
+    SleepBefore: {^TimeSpec: {value: 0, unit: seconds}}
+  - To: Update
+    Weight: 0
+    SleepBefore: {^TimeSpec: {value: 0, unit: seconds}}
+  
+
+
+Encryption:
+  
+  UseCryptSharedLib: true
+  CryptSharedLibPath: /data/workdir/mongocrypt/lib/mongo_crypt_v1.so
+  
+  EncryptedCollections:
+  - Database: *encrypted_db
+    Collection: *encrypted_coll
+    EncryptionType: queryable
+
+    QueryableEncryptedFields:
+      tm_retail_tx: { type: "long", queries: [{queryType: "range", min: 0, max: 1000000000000, contention: 4, sparsity: 2, trimFactor: 8}] }
+      age_hospitals: { type: "int",  queries: [{queryType: "range", min: 0, max: 55000, contention: 8, sparsity: 2, trimFactor: 6}] }
+      bnk_bal: { type: "decimal", queries: [{queryType: "range", min: 0, max: 1000000, precision: 2, contention: 8, sparsity: 2, trimFactor: 6}] }
+
+
+Clients:
+  EncryptedPool:
+    QueryOptions:
+      maxPoolSize: 400
+  
+    EncryptionOptions:
+      KeyVaultDatabase: "keyvault"
+      KeyVaultCollection: "datakeys"
+      EncryptedCollections:
+      - genny_release_criteria.my_coll
+  
+
+Actors:
+
+  
+  
+  - Name: InsertActor_Thread0
```

Comment:
Can we template / parameterize these insert actors the only changes are the startFromLine AFAICS.

See template definition [here](https://github.com/mongodb/genny/blob/f97023de49be506ec51b49f7caaecb85a1835d24/src/phases/query/CollScanSimplifiablePredicate.yml#L14-L31) and usage [here](https://github.com/mongodb/genny/blob/f97023de49be506ec51b49f7caaecb85a1835d24/src/phases/query/CollScanSimplifiablePredicate.yml#L459-L477)

Reply:
I've done some templatization here, let me know if that is what you're looking for.